modules/features/podcasts/src/main/java/au/com/shiftyjelly/pocketcasts/podcasts/view/components/ratings/StarRatingView.kt

```diff
@@ -62,9 +64,6 @@ private fun Content(
     state: RatingState.Loaded,
     onClick: () -> Unit
 ) {
-    if (state.noRatings) {
```

Comment:
`noRatings` in `Loaded` state is redundant now and can be removed.

Reply:
Good catch. Removed in https://github.com/Automattic/pocket-casts-android/pull/1453/commits/23770de8cc1f4c950b919d9ea45f3a4cbd093a35